<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #0E84FC;
	color: #fff;
	border-radius: 5px;
	padding: 10px;
}
QPushButton:focus {
	background-color: #015EBD;
	outline: none;
}
QCheckBox {
	color: #fff;
	font: 500 11pt &quot;Montserrat SemiBold&quot;;
}
QCheckBox::hover {
	color: #0e84fc;
	font: 500 11pt &quot;Montserrat SemiBold&quot;;
}
QPushButton::hover {
	background-color: #015EBD;
}
QProgressBar::chunk  {
	color: #fff;
	background-color: #015EBD;
}
QToolButton {
	color: #fff;
}
QLabel {
	color: #fff;
	font: 600 10pt &quot;Montserrat SemiBold&quot;;
}
QLabel::hover {
	color: #015EBD;
}
QLineEdit {
	border-radius: 5px;
	padding: 5px  5px  7px  8px ;
	border: 1px solid #353535;
	font: 600 9pt &quot;Montserrat SemiBold&quot;;
}
QRadioButton {
	color: #fff;
	font: 600 10pt &quot;Montserrat SemiBold&quot;;
}
QRadioButton::hover {
	color: #015EBD;
}
QComboBox {
	color: #000;
	font: 600 11pt &quot;Montserrat SemiBold&quot;;
	background-color: #fff;
	border-radius: 5px;
	padding: 10px;
}
QComboBox::drop-down {
	margin: -3px 5px 7px -40px;
	border-radius: 0px;
	background-image: url(:/newPrefix/down.png);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color:  #353535;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>179</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: #fff;
	font: 600 14pt &quot;Montserrat SemiBold&quot;;
}</string>
            </property>
            <property name="text">
             <string>Prog envio de e-mail</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>179</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color:  #353535;
}
QTextEdit {
	background-color: white;
	border-radius: 5px;
	padding: 5px  5px  7px  8px ;
	border: 1px solid #353535;
	font: 600 9pt &quot;Montserrat SemiBold&quot;;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_dest">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
            <property name="placeholderText">
             <string>Destinatario</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_title">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
            <property name="placeholderText">
             <string>Assunto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="le_text">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat SemiBold'; font-size:9pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="placeholderText">
             <string>Corpo do email</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_email">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
            <property name="placeholderText">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_smtp">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
            <property name="placeholderText">
             <string>Servidor SMTP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_passwd">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>64</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Senha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_port">
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
            <property name="placeholderText">
             <string>Porta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_ssl">
            <property name="text">
             <string>Utiliza SSL</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color:  #353535;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="lb_result">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_send">
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
